lib/options.json

```diff
@@ -140,7 +140,14 @@
   },
   "properties": {
     "bonjour": {
-      "type": "boolean",
+      "anyOf": [
+        {
+          "type": "boolean"
+        },
+        {
+          "type": "object"
+        }
```

Comment:
Need to add `"properties"` as well. https://www.npmjs.com/package/bonjour#var-service--bonjourpublishoptions

Reply:
Could be problematic if they change in the package, we'll have to update manually and do release everytime

Reply:
I would prefer to keep it as `boolean` only and respect `options.https`

Reply:
It also seems this package doesn't go under changes anymore. Last publish 5 years ago.

Reply:
we need to support all options

Reply:
hm then we can have properties, lets do it